src/main/java/com/github/onsdigital/babbage/locale/LocaleConfig.java

```diff
@@ -31,43 +20,26 @@ public class LocaleConfig {
      * @return
      */
     public static Map<String, String> getLabels(Locale locale) {
-// disabled caching while in development
-//        if (!localeToLabels.containsKey(locale))
-//            synchronized (LocaleConfig.class) {
-//                if (!localeToLabels.containsKey(locale)) {
-//                    // resource bundle class used under the covers to automate the selection of the bundle file for the given
-//                    // locale. The supported locales are defined by the available bundle property files.
-//                    Properties properties = loadProperties(locale);
-//                    Map<String, String> labels = toMap(properties);
-//                    localeToLabels.put(locale, labels);
-//                }
-//            }
-//
-//        return localeToLabels.get(locale);
-
         Properties properties = loadProperties(locale);
         Map<String, String> labels = toMap(properties);
         return labels;
     }
-
+ 
     private static Properties loadProperties(Locale locale) {
-        Properties properties;
-
-        try {
+        if (!getDefaultLocale().equals(locale)) {
             try {
-                if (locale.equals(english)) {
-                    properties = LoadProperties("LabelsBundle.properties");
-                } else {
-                    properties = LoadProperties("LabelsBundle_" + locale.getLanguage() + ".properties");
-                }
-
+                return LoadProperties("LabelsBundle_" + locale.getLanguage() + ".properties");
             } catch (IOException e) {
-                properties = LoadProperties("LabelsBundle.properties");
+                // Error finding properties. No problem, let's try the default language...
             }
+        }
+
+        // Default language (English)
+        try {
+            return LoadProperties("LabelsBundle.properties");
         } catch (IOException e) {
             throw new RuntimeException("Failed to load search properties file", e);
```

Comment:
Not your mistake, but the error text is misleading
```suggestion
            throw new RuntimeException("failed to load LabelsBundle properties file", e);
```